backend/app/tests/anti_recommendation_engine/conftest.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from pytest_mock import MockFixture
+
+from app.anti_recommenders.open_ai.normal_open_ai_anti_recommender import (
+    NormalOpenAiAntiRecommender,
+)
+from app.models.settings import settings
+from app.models.types import AntiRecommenderType
+
+ANTI_RECOMMENDER_TYPES = frozenset([AntiRecommenderType.OPEN_AI])
+
+
+@pytest.fixture(autouse=True, params=ANTI_RECOMMENDER_TYPES, scope="module")
```

Comment:
I've never seen this autouse bit before, I can definitely use that for global configuration in some of my projects.